docs/metrics/getting-started-histogram/README.md

```diff
@@ -34,17 +34,17 @@ output from the console, similar to shown below:
 ```text
 Export 14:30:58.201 14:30:59.177 histogram [tag1=value1;tag2=value2] Histogram, Meter: TestMeter/0.0.1
```

Comment:
The timestamp is updated to use the [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) UTC timestamp, so the output would look like:

```text
Export (2021-09-02T00:47:54.9655496Z, 2021-09-02T00:48:00.0963511Z] histogram tag1=value1;tag2=value2 Histogram, Meter: TestMeter/0.0.1
Value: Sum: 159160 Count: 313
(-Infinity, 0]: 0
(0, 5]: 2
(5, 10]: 1
(10, 25]: 5
(25, 50]: 7
(50, 75]: 5
(75, 100]: 8
(100, 250]: 40
(250, 500]: 90
(500, 1000]: 155
(1000, +Infinity): 0
```

Reply:
Thanks for pointing me to the reference! 
I did a copy paste from console this time, should be accurate now. :-)